sorc/ncep_post.fd/SURFCE.f

```diff
@@ -6509,7 +5889,20 @@ SUBROUTINE SURFCE
          if (allocated(rcs))    deallocate(rcs)
          if (allocated(gc))     deallocate(gc)
 
-
+!---------
+!	 print*,'outputting leaf area index= ',XLAI
+         IF (IGET(254).GT.0 )THEN
+          DO J=JSTA,JEND
+           DO I=1,IM
+             GRID1(I,J) = XLAI
+           ENDDO
+          ENDDO
+           if(grib=='grib2') then
+            cfld=cfld+1
+            fld_info(cfld)%ifld=IAVBLFLD(IGET(254))
+            datapd(1:im,1:jend-jsta+1,cfld)=GRID1(1:im,jsta:jend)
```

Comment:
@JesseMeng-NOAA This part should be synced with upsteam/develop.

Reply:
Fixed. Thanks